### DISCUSSION TITLE
WalletConnect Android SDK for Java (not Kotlin) current version

### DISCUSSION THREAD

**Discussion by andrewjamesnesbitt** [WalletConnect/walletconnect-monorepo#4115] [Developer Support]
Is there a WalletConnect Android SDK for Java (not Kotlin)? If so is there a worked simple example integrating a wallet (PERA) with a native Android App?

**Comment by glitch-txs** [comment]
Hi, we don't support Java at the moment, only Kotlin or Flutter
